tools/m1_utils/update_static.sh

```diff
@@ -12,11 +12,12 @@ unarchive() {
 
     # Assume that if unarchiving fails, it's already an object.
```

Comment:
Note: this comment is obsolete now as we've got fat object archive specifically handled below